Makefile

```diff
@@ -66,6 +66,10 @@ clean:
 # generate project
 #################################################################
 
+overwrite:
+	rm -rf $(COOKIECUTTER_OUTPUT_DIR)/behave* $(COOKIECUTTER_OUTPUT_DIR)/features
+	poetry run cookiecutter --overwrite-if-exists --default-config --no-input -o $(BUILD_DIR) .
+
 refresh: clean
 	poetry run cookiecutter --overwrite-if-exists --default-config --no-input -o $(BUILD_DIR) .
```

Comment:
Remove `--overwrite-if-exists` to explain why there are 2 targets (refresh and overwrite)

Reply:
done